reg_tests/test_files/vofWaveGenerator/vofWaveGenerator.i

```diff
@@ -237,11 +237,12 @@ realms:
       use_consolidated_face_elem_bc_algorithm: yes
 
       activate_balanced_force_algorithm: yes
+      activate_buoyancy_pressure_stabilization: yes
```

Comment:
This was to be perfectly explicit, as when BF is active, buoyancy pressure stabilization is also (similar to vofBuoy)... Keep this as I like the inclusion of this line command.